### PR TITLE
Polyfill match api syntax

### PR DESCRIPTION
**Pull request by jihyerish**
- Match the syntax of Element APIs with Spec
  - `Element.focusableAreas`, `Element.getSpatnavContainer`
  - Get rid of `findCandidates` function which isn't used.
- Change the way of setting the standard syntax option

### CODE REVIEW

**Comment by jihyerish** [comment]
@frivoal Actually, I'm thinking about merging spatnav-heuristic.js and spatnav-api.js into one JS file.
Do you think this is a good approach?

**Comment by frivoal** [comment]
> I'm thinking about merging spatnav-heuristic.js and spatnav-api.js into one JS file.
Do you think this is a good approach?

Yes. You can keep the code separated internally, but a single file is easier to use.

**Comment by jihyerish** [comment]
I've changed the README, so I'll merge this PR.
About merging spatnav-heuristic.js and spatnav-api.js into a single file, I'll make a new PR.

File: polyfill/spatnav-heuristic.js [hunk 1]
```diff
@@ -441,15 +413,41 @@ function focusNavigationHeuristics() {
...
 
           if(Array.isArray(recursiveFocusables) && recursiveFocusables.length){
             focusables = focusables.concat(recursiveFocusables);
           }
         }
       }
     }
-    return focusables;
+
+    if (option === 'all') {
```
Comment (frivoal) [inline polyfill/spatnav-heuristic.js]: `option` should not directly be a string equal to `"all"` or `"visible"`, but an object with a `mode` member, and `mode` is a string equal to `"all"` or `"visible"`.

Comment (jihyerish) [reply polyfill/spatnav-heuristic.js]: Thanks for the comment! I've fixed this.

**Comment by jihyerish** [comment]
@frivoal I've made the change based on your comment. Could you check the update?

**Pull request merged**